app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
```

Comment:
Why do you have App\Blog here but you do not have Blog.php under App directory.

Reply:
Fixes issue.